site/src/util/mdxComponents.tsx

```diff
@@ -1,10 +1,37 @@
+/* eslint-disable react-hooks/rules-of-hooks */
```

Comment:
It might be safer to replace this comment with `// eslint-disable-next-line react-hooks/rules-of-hooks` called three times. Otherwise, we can miss real bugs in the future such as conditional hook calls after early exits.

I can also imagine us splitting the lookup as it has grew too big. If we do that, we won't have to silence the rule.

Reply:
Agreed, addressed this in https://github.com/blockprotocol/blockprotocol/pull/65/commits/9100cac4862dabd0bd2c3cf1b6ebae7cb8377696